### PR TITLE
Update Workflow/Activity Worker wording

### PR DESCRIPTION
**Pull request by lorensr**
If I don't include workflows/activities, the workflow/activity worker won't run

### CODE REVIEW

**Comment by cretz** [comment]
> If I don't include workflows/activities, the workflow/activity worker won't run

This is not true in every SDK. But that statement doesn't seem to be in the docs, so LGTM.

File: docs-src/concepts/what-is-a-worker-entity.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ tags:
 A Worker Entity is the individual Worker within a Worker Process that listens to a specific Task Queue.
 
 A Worker Entity listens and polls on a single Task Queue.
-A Worker Entity contains both a Workflow Worker and an Activity Worker so that it may make progress of either a Workflow Execution or an Activity Execution.
+A Worker Entity contains a Workflow Worker and/or an Activity Worker, which make progress on Workflow Executions and Activity Executions, respectively.
```
Comment (flossypurse) [inline docs-src/concepts/what-is-a-worker-entity.md:14]:
```suggestion
A Worker Entity contains a Workflow Worker and/or an Activity Worker, which makes progress on Workflow Executions and Activity Executions, respectively.
```

File: docs-src/java/how-to-develop-a-worker-program-in-java.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ A [Worker Entity](/concepts/what-is-a-worker-entity) is the component within a W
 Although multiple Worker Entities can be in a single Worker Process, a single Worker Entity Worker Process may be perfectly sufficient.
 For more information, see the [Worker tuning guide](/dev-guide/worker-performance).
 
-A Worker Entity contains both a Workflow Worker and an Activity Worker so that it can make progress for either a Workflow Execution or an Activity Execution.
+A Worker Entity contains a Workflow Worker and/or an Activity Worker, which make progress on Workflow Executions and Activity Executions, respectively.
```
Comment (flossypurse) [inline docs-src/java/how-to-develop-a-worker-program-in-java.md:23]:
```suggestion
A Worker Entity contains a Workflow Worker and/or an Activity Worker, which makes progress on Workflow Executions and Activity Executions, respectively.
```

File: docs-src/php/run-a-dev-worker.md [hunk 3]
```diff
@@ -18,4 +18,4 @@ A [Worker Entity](/concepts/what-is-a-worker-entity) is the component within a W
 Although multiple Worker Entities can be in a single Worker Process, a single Worker Entity Worker Process may be perfectly sufficient.
 For more information, see the [Worker tuning guide](/dev-guide/worker-performance).
 
-A Worker Entity contains both a Workflow Worker and an Activity Worker so that it can make progress for either a Workflow Execution or an Activity Execution.
+A Worker Entity contains a Workflow Worker and/or an Activity Worker, which make progress on Workflow Executions and Activity Executions, respectively.
```
Comment (flossypurse) [inline docs-src/php/run-a-dev-worker.md:21]:
```suggestion
A Worker Entity contains a Workflow Worker and/or an Activity Worker, which makes progress on Workflow Executions and Activity Executions, respectively.
```

File: docs-src/python/run-a-dev-worker.md [hunk 4]
```diff
@@ -18,4 +18,4 @@ A [Worker Entity](/concepts/what-is-a-worker-entity) is the component within a W
 Although multiple Worker Entities can be in a single Worker Process, a single Worker Entity Worker Process may be perfectly sufficient.
 For more information, see the [Worker tuning guide](/dev-guide/worker-performance).
 
-A Worker Entity contains both a Workflow Worker and an Activity Worker so that it can make progress for either a Workflow Execution or an Activity Execution.
+A Worker Entity contains a Workflow Worker and/or an Activity Worker, which make progress on Workflow Executions and Activity Executions, respectively.
```
Comment (flossypurse) [inline docs-src/python/run-a-dev-worker.md:21]:
```suggestion
A Worker Entity contains a Workflow Worker and/or an Activity Worker, which makes progress on Workflow Executions and Activity Executions, respectively.
```

File: docs-src/typescript/run-a-dev-worker.md [hunk 5]
```diff
@@ -18,4 +18,4 @@ A [Worker Entity](/concepts/what-is-a-worker-entity) is the component within a W
 Although multiple Worker Entities can be in a single Worker Process, a single Worker Entity Worker Process may be perfectly sufficient.
 For more information, see the [Worker tuning guide](/dev-guide/worker-performance).
 
-A Worker Entity contains both a Workflow Worker and an Activity Worker so that it can make progress for either a Workflow Execution or an Activity Execution.
+A Worker Entity contains a Workflow Worker and/or an Activity Worker, which make progress on Workflow Executions and Activity Executions, respectively.
```
Comment (flossypurse) [inline docs-src/typescript/run-a-dev-worker.md:21]:
```suggestion
A Worker Entity contains a Workflow Worker and/or an Activity Worker, which makes progress on Workflow Executions and Activity Executions, respectively.
```

**Review by flossypurse — APPROVED** [review]

**Pull request merged**